### ISSUE TITLE
VM live migration --> Pacemaker unstable

### ISSUE THREAD

**Issue by insatomcat** [seapath/meta-seapath#25]
**When trying live migration**, since there is a problem (see #22), rebooting a seapath host is stuck by pacemaker waiting for a timeout during shutdown ("a stop job is running...") and delaying the reboot process, up to systemd configured timeout TimeoutStopSec (originaly 30min but I'm currently testing with 3min).
The VM which were running on that node all go "FAILED" (with an extra "blocked" status) and do no migrate, unless I issue a "crm resource cleanup" order:

```
14 resource instances configured (4 BLOCKED from further action due to failure)
...
  * VM3  (ocf::heartbeat:VirtualDomain):  FAILED hypervisor3 (blocked)
...
```

```
Failed Resource Actions:
  * VM1_stop_0 on hypervisor3 'error' (1): call=78, status='complete', exitreason='forced stop failed', last-rc-change='2021-11-29 11:46:48Z', queued=0ms, exec=2189ms
  * VM2_stop_0 on hypervisor3 'error' (1): call=79, status='complete', exitreason='forced stop failed', last-rc-change='2021-11-29 11:46:48Z', queued=0ms, exec=2189ms
  * VM3_stop_0 on hypervisor3 'error' (1): call=80, status='complete', exitreason='forced stop failed', last-rc-change='2021-11-29 11:46:48Z', queued=0ms, exec=2188ms
  * VM4_stop_0 on hypervisor3 'error' (1): call=81, status='complete', exitreason='forced stop failed', last-rc-change='2021-11-29 11:46:48Z', queued=0ms, exec=2187ms
```

I do not understand why the host does not reboot cleanly.
And why pacemaker marks the VM as failed and does not migrate them to the other hypervisor automatically.

Note that I also tested by changing the "WantedBy":

```
[Install]
WantedBy=corosync.service
```

It does not help...

I'm guessing this problem may go away if we solve #22 
Thanks.

This is the shutdown log:

```
-- Logs begin at Mon 2021-11-29 11:49:11 UTC, end at Mon 2021-11-29 12:44:12 UTC. --
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target Libvirt guests shutdown.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopping Virtualization daemon...
Nov 29 12:34:47 hypervisor3 systemd[1]: libvirtd.service: Succeeded.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped Virtualization daemon.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target Remote File Systems.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target Remote File Systems (Pre).
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopping Map RBD devices...
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopping Login Service...
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopping Virtual machine log manager...
Nov 29 12:34:47 hypervisor3 systemd[1]: virtlogd.service: Succeeded.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped Virtual machine log manager.
Nov 29 12:34:47 hypervisor3 rbdmap[5443]: No /etc/ceph/rbdmap found.
Nov 29 12:34:47 hypervisor3 systemd[1]: rbdmap.service: Succeeded.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped Map RBD devices.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target ceph target allowing to start/stop all ceph*@.service instances at once.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target ceph target allowing to start/stop all ceph-mds@.service instances at once.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target ceph target allowing to start/stop all ceph-mgr@.service instances at once.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target ceph target allowing to start/stop all ceph-osd@.service instances at once.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target ceph target allowing to start/stop all ceph-radosgw@.service instances at once.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target ceph target allowing to start/stop all ceph-mon@.service instances at once.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped target ceph target allowing to start/stop all ceph-rbd-mirror@.service instances at once.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopping Ceph crash dump collector...
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopping Ceph cluster manager daemon...
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopping Ceph cluster monitor daemon...
Nov 29 12:34:47 hypervisor3 ceph-mon[965]: 2021-11-29 12:34:47.139 7c924e00b700 -1 received  signal: Terminated from /sbin/init extra_kernel_parameters_markers  (PID: 1) UID: 0
Nov 29 12:34:47 hypervisor3 ceph-mon[965]: 2021-11-29 12:34:47.139 7c924e00b700 -1 mon.hypervisor3@2(peon) e1 *** Got Signal Terminated ***
Nov 29 12:34:47 hypervisor3 systemd[1]: ceph-crash.service: Succeeded.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped Ceph crash dump collector.
Nov 29 12:34:47 hypervisor3 systemd[1]: systemd-logind.service: Succeeded.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped Login Service.
Nov 29 12:34:47 hypervisor3 systemd[1]: ceph-mgr@hypervisor3.service: Succeeded.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped Ceph cluster manager daemon.
Nov 29 12:34:47 hypervisor3 systemd[1]: Removed slice system-ceph\x2dmgr.slice.
Nov 29 12:34:47 hypervisor3 systemd[1]: ceph-mon@hypervisor3.service: Succeeded.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped Ceph cluster monitor daemon.
Nov 29 12:34:47 hypervisor3 systemd[1]: Removed slice system-ceph\x2dmon.slice.
Nov 29 12:34:47 hypervisor3 systemd[1]: systemd-random-seed.service: Succeeded.
Nov 29 12:34:47 hypervisor3 systemd[1]: Stopped Load/Save Random Seed.
Nov 29 12:34:47 hypervisor3 VirtualDomain(SELOG)[5455]: ERROR: SELOG: live migration to hypervisor1 failed: 1
Nov 29 12:34:47 hypervisor3 VirtualDomain(SCADA)[5461]: ERROR: SCADA: live migration to hypervisor2 failed: 1
Nov 29 12:34:47 hypervisor3 VirtualDomain(centos7SP1)[5462]: ERROR: centos7SP1: live migration to hypervisor2 failed: 1
Nov 29 12:34:47 hypervisor3 VirtualDomain(AHYPE)[5464]: ERROR: AHYPE: live migration to hypervisor1 failed: 1
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: SELOG_migrate_to_0:5177:stderr [ error: Disconnected from qemu:///system due to end of file ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: SELOG_migrate_to_0:5177:stderr [ error: Cannot recv data: Host key verification failed.: Connection reset by peer ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: SELOG_migrate_to_0:5177:stderr [ ocf-exit-reason:SELOG: live migration to hypervisor1 failed: 1 ]
Nov 29 12:34:47 hypervisor3 pacemaker-controld[1008]:  notice: Result of migrate_to operation for SELOG on hypervisor3: 1 (error)
Nov 29 12:34:47 hypervisor3 pacemaker-controld[1008]:  notice: hypervisor3-SELOG_migrate_to_0:71 [ error: Disconnected from qemu:///system due to end of file\nerror: Cannot recv data: Host key verification failed.: Connection reset by peer\nocf-exit-reason:SELOG: live migration to hypervisor1 failed: 1\n ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: SCADA_migrate_to_0:5175:stderr [ error: Disconnected from qemu:///system due to end of file ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: SCADA_migrate_to_0:5175:stderr [ error: Cannot recv data: Host key verification failed.: Connection reset by peer ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: SCADA_migrate_to_0:5175:stderr [ ocf-exit-reason:SCADA: live migration to hypervisor2 failed: 1 ]
Nov 29 12:34:47 hypervisor3 pacemaker-controld[1008]:  notice: Result of migrate_to operation for SCADA on hypervisor3: 1 (error)
Nov 29 12:34:47 hypervisor3 pacemaker-controld[1008]:  notice: hypervisor3-SCADA_migrate_to_0:69 [ error: Disconnected from qemu:///system due to end of file\nerror: Cannot recv data: Host key verification failed.: Connection reset by peer\nocf-exit-reason:SCADA: live migration to hypervisor2 failed: 1\n ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: AHYPE_migrate_to_0:5174:stderr [ error: Disconnected from qemu:///system due to end of file ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: AHYPE_migrate_to_0:5174:stderr [ error: Cannot recv data: Host key verification failed.: Connection reset by peer ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: AHYPE_migrate_to_0:5174:stderr [ ocf-exit-reason:AHYPE: live migration to hypervisor1 failed: 1 ]
Nov 29 12:34:47 hypervisor3 pacemaker-controld[1008]:  notice: Result of migrate_to operation for AHYPE on hypervisor3: 1 (error)
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: centos7SP1_migrate_to_0:5178:stderr [ error: Disconnected from qemu:///system due to end of file ]
Nov 29 12:34:47 hypervisor3 pacemaker-controld[1008]:  notice: hypervisor3-AHYPE_migrate_to_0:67 [ error: Disconnected from qemu:///system due to end of file\nerror: Cannot recv data: Host key verification failed.: Connection reset by peer\nocf-exit-reason:AHYPE: live migration to hypervisor1 failed: 1\n ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: centos7SP1_migrate_to_0:5178:stderr [ error: Cannot recv data: Host key verification failed.: Connection reset by peer ]
Nov 29 12:34:47 hypervisor3 pacemaker-execd[1005]:  notice: centos7SP1_migrate_to_0:5178:stderr [ ocf-exit-reason:centos7SP1: live migration to hypervisor2 failed: 1 ]
Nov 29 12:34:47 hypervisor3 pacemaker-controld[1008]:  notice: Result of migrate_to operation for centos7SP1 on hypervisor3: 1 (error)
Nov 29 12:34:47 hypervisor3 pacemaker-controld[1008]:  notice: hypervisor3-centos7SP1_migrate_to_0:73 [ error: Disconnected from qemu:///system due to end of file\nerror: Cannot recv data: Host key verification failed.: Connection reset by peer\nocf-exit-reason:centos7SP1: live migration to hypervisor2 failed: 1\n ]
Nov 29 12:34:47 hypervisor3 VirtualDomain(SCADA)[5605]: INFO: Virtual domain SCADA currently has no state, retrying.
Nov 29 12:34:47 hypervisor3 VirtualDomain(centos7SP1)[5612]: INFO: Virtual domain centos7SP1 currently has no state, retrying.
Nov 29 12:34:47 hypervisor3 VirtualDomain(AHYPE)[5622]: INFO: Virtual domain AHYPE currently has no state, retrying.
Nov 29 12:34:47 hypervisor3 VirtualDomain(SELOG)[5623]: INFO: Virtual domain SELOG currently has no state, retrying.
Nov 29 12:34:48 hypervisor3 VirtualDomain(centos7SP1)[5655]: INFO: Virtual domain centos7SP1 currently has no state, retrying.
Nov 29 12:34:48 hypervisor3 VirtualDomain(SCADA)[5654]: INFO: Virtual domain SCADA currently has no state, retrying.
Nov 29 12:34:48 hypervisor3 VirtualDomain(SELOG)[5657]: INFO: Virtual domain SELOG currently has no state, retrying.
Nov 29 12:34:48 hypervisor3 VirtualDomain(AHYPE)[5659]: INFO: Virtual domain AHYPE currently has no state, retrying.
Nov 29 12:34:49 hypervisor3 VirtualDomain(centos7SP1)[5710]: ERROR: Virtual domain centos7SP1 has no state during stop operation, bailing out.
Nov 29 12:34:49 hypervisor3 VirtualDomain(SCADA)[5713]: ERROR: Virtual domain SCADA has no state during stop operation, bailing out.
Nov 29 12:34:49 hypervisor3 VirtualDomain(SELOG)[5711]: ERROR: Virtual domain SELOG has no state during stop operation, bailing out.
Nov 29 12:34:49 hypervisor3 VirtualDomain(AHYPE)[5718]: ERROR: Virtual domain AHYPE has no state during stop operation, bailing out.
Nov 29 12:34:49 hypervisor3 VirtualDomain(SELOG)[5732]: INFO: Issuing forced shutdown (destroy) request for domain SELOG.
Nov 29 12:34:49 hypervisor3 VirtualDomain(centos7SP1)[5734]: INFO: Issuing forced shutdown (destroy) request for domain centos7SP1.
Nov 29 12:34:49 hypervisor3 VirtualDomain(SCADA)[5738]: INFO: Issuing forced shutdown (destroy) request for domain SCADA.
Nov 29 12:34:49 hypervisor3 VirtualDomain(AHYPE)[5741]: INFO: Issuing forced shutdown (destroy) request for domain AHYPE.
Nov 29 12:34:49 hypervisor3 VirtualDomain(AHYPE)[5787]: ERROR: forced stop failed
Nov 29 12:34:49 hypervisor3 VirtualDomain(SCADA)[5789]: ERROR: forced stop failed
Nov 29 12:34:49 hypervisor3 VirtualDomain(centos7SP1)[5788]: ERROR: forced stop failed
Nov 29 12:34:49 hypervisor3 VirtualDomain(SELOG)[5790]: ERROR: forced stop failed
Nov 29 12:34:49 hypervisor3 pacemaker-execd[1005]:  notice: SCADA_stop_0:5480:stderr [ error: failed to connect to the hypervisor error: failed to connect socket to '/var/run/libvirt/libvirt-sock': no such file or directory ]
Nov 29 12:34:49 hypervisor3 pacemaker-execd[1005]:  notice: SCADA_stop_0:5480:stderr [ ocf-exit-reason:forced stop failed ]
Nov 29 12:34:49 hypervisor3 pacemaker-controld[1008]:  notice: Result of stop operation for SCADA on hypervisor3: 1 (error)
Nov 29 12:34:49 hypervisor3 pacemaker-controld[1008]:  notice: hypervisor3-SCADA_stop_0:75 [ error: failed to connect to the hypervisor error: failed to connect socket to '/var/run/libvirt/libvirt-sock': no such file or directory\nocf-exit-reason:forced stop failed\n ]
Nov 29 12:34:49 hypervisor3 pacemaker-execd[1005]:  notice: centos7SP1_stop_0:5483:stderr [ error: failed to connect to the hypervisor error: failed to connect socket to '/var/run/libvirt/libvirt-sock': no such file or directory ]
Nov 29 12:34:49 hypervisor3 pacemaker-execd[1005]:  notice: centos7SP1_stop_0:5483:stderr [ ocf-exit-reason:forced stop failed ]
Nov 29 12:34:49 hypervisor3 pacemaker-controld[1008]:  notice: Result of stop operation for centos7SP1 on hypervisor3: 1 (error)
Nov 29 12:34:49 hypervisor3 pacemaker-controld[1008]:  notice: hypervisor3-centos7SP1_stop_0:77 [ error: failed to connect to the hypervisor error: failed to connect socket to '/var/run/libvirt/libvirt-sock': no such file or directory\nocf-exit-reason:forced stop failed\n ]
Nov 29 12:34:49 hypervisor3 pacemaker-execd[1005]:  notice: SELOG_stop_0:5481:stderr [ error: failed to connect to the hypervisor error: failed to connect socket to '/var/run/libvirt/libvirt-sock': no such file or directory ]
Nov 29 12:34:49 hypervisor3 pacemaker-execd[1005]:  notice: SELOG_stop_0:5481:stderr [ ocf-exit-reason:forced stop failed ]
Nov 29 12:34:49 hypervisor3 pacemaker-execd[1005]:  notice: AHYPE_stop_0:5479:stderr [ error: failed to connect to the hypervisor error: failed to connect socket to '/var/run/libvirt/libvirt-sock': no such file or directory ]
Nov 29 12:34:49 hypervisor3 pacemaker-execd[1005]:  notice: AHYPE_stop_0:5479:stderr [ ocf-exit-reason:forced stop failed ]
Nov 29 12:34:49 hypervisor3 pacemaker-controld[1008]:  notice: Result of stop operation for SELOG on hypervisor3: 1 (error)
Nov 29 12:34:49 hypervisor3 pacemaker-controld[1008]:  notice: hypervisor3-SELOG_stop_0:76 [ error: failed to connect to the hypervisor error: failed to connect socket to '/var/run/libvirt/libvirt-sock': no such file or directory\nocf-exit-reason:forced stop failed\n ]
Nov 29 12:34:49 hypervisor3 pacemaker-controld[1008]:  notice: Result of stop operation for AHYPE on hypervisor3: 1 (error)
Nov 29 12:34:49 hypervisor3 pacemaker-controld[1008]:  notice: hypervisor3-AHYPE_stop_0:74 [ error: failed to connect to the hypervisor error: failed to connect socket to '/var/run/libvirt/libvirt-sock': no such file or directory\nocf-exit-reason:forced stop failed\n ]
Nov 29 12:34:49 hypervisor3 pacemaker-attrd[1006]:  notice: Setting fail-count-SCADA#stop_0[hypervisor3]: (unset) -> INFINITY
Nov 29 12:34:49 hypervisor3 pacemaker-attrd[1006]:  notice: Setting last-failure-SCADA#stop_0[hypervisor3]: (unset) -> 1638189289
Nov 29 12:34:49 hypervisor3 pacemaker-attrd[1006]:  notice: Setting fail-count-centos7SP1#stop_0[hypervisor3]: (unset) -> INFINITY
Nov 29 12:34:49 hypervisor3 pacemaker-attrd[1006]:  notice: Setting last-failure-centos7SP1#stop_0[hypervisor3]: (unset) -> 1638189289
Nov 29 12:34:49 hypervisor3 pacemaker-attrd[1006]:  notice: Setting fail-count-SELOG#stop_0[hypervisor3]: (unset) -> INFINITY
Nov 29 12:34:49 hypervisor3 pacemaker-attrd[1006]:  notice: Setting last-failure-SELOG#stop_0[hypervisor3]: (unset) -> 1638189289
Nov 29 12:34:49 hypervisor3 pacemaker-attrd[1006]:  notice: Setting fail-count-AHYPE#stop_0[hypervisor3]: (unset) -> INFINITY
Nov 29 12:34:49 hypervisor3 pacemaker-attrd[1006]:  notice: Setting last-failure-AHYPE#stop_0[hypervisor3]: (unset) -> 1638189289
Nov 29 12:37:46 hypervisor3 systemd[1]: pacemaker.service: State 'stop-sigterm' timed out. Skipping SIGKILL.
Nov 29 12:37:46 hypervisor3 pacemakerd[989]:  notice: Caught 'Terminated' signal
Nov 29 12:37:46 hypervisor3 pacemakerd[989]:  notice: Still waiting for pacemaker-controld to terminate
Nov 29 12:40:47 hypervisor3 systemd[1]: pacemaker.service: State 'stop-final-sigterm' timed out. Skipping SIGKILL. Entering failed mode.
Nov 29 12:40:47 hypervisor3 systemd[1]: pacemaker.service: Failed with result 'timeout'.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Pacemaker High Availability Cluster Manager.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target resource-agents dependencies.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target System Time Synchronized.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target System Time Set.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopping Corosync Cluster Engine...
Nov 29 12:40:47 hypervisor3 corosync[966]:  [MAIN  ] Node was shut down by a signal
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopping D-Bus System Message Bus...
Nov 29 12:40:47 hypervisor3 systemd[1]: dbus.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped D-Bus System Message Bus.
Nov 29 12:40:47 hypervisor3 corosync[966]:  [SERV  ] Unloading all Corosync service engines.
Nov 29 12:40:47 hypervisor3 corosync[966]:  [SERV  ] Service engine unloaded: corosync vote quorum service v1.0
Nov 29 12:40:47 hypervisor3 corosync[966]:  [SERV  ] Service engine unloaded: corosync configuration map access
Nov 29 12:40:47 hypervisor3 pacemakerd[989]:  error: Connection destroyed
Nov 29 12:40:47 hypervisor3 pacemakerd[989]:  notice: Still waiting for pacemaker-controld to terminate
Nov 29 12:40:47 hypervisor3 corosync[966]:  [SERV  ] Service engine unloaded: corosync configuration service
Nov 29 12:40:47 hypervisor3 pacemaker-controld[1008]:  error: Connection to the CPG API failed: Library error (2)
Nov 29 12:40:47 hypervisor3 pacemaker-controld[1008]:  crit: Lost connection to cluster layer, shutting down
Nov 29 12:40:47 hypervisor3 pacemaker-controld[1008]:  notice: Forcing immediate exit with status 102 (Not connected)
Nov 29 12:40:47 hypervisor3 pacemaker-attrd[1006]:  error: Connection to the CPG API failed: Library error (2)
Nov 29 12:40:47 hypervisor3 pacemaker-attrd[1006]:  crit: Lost connection to cluster layer, shutting down
Nov 29 12:40:47 hypervisor3 pacemaker-based[1003]:  error: Connection to the CPG API failed: Library error (2)
Nov 29 12:40:47 hypervisor3 pacemaker-based[1003]:  crit: Lost connection to cluster layer, shutting down
Nov 29 12:40:47 hypervisor3 pacemaker-fenced[1004]:  error: Connection to cib_rw failed
Nov 29 12:40:47 hypervisor3 pacemaker-fenced[1004]:  error: Connection to cib_rw closed
Nov 29 12:40:47 hypervisor3 pacemaker-fenced[1004]:  crit: Lost connection to the CIB manager, shutting down
Nov 29 12:40:47 hypervisor3 pacemakerd[989]:  error: pacemaker-controld[1008] exited with status 102 (Not connected)
Nov 29 12:40:47 hypervisor3 pacemakerd[989]:  notice: Stopping pacemaker-schedulerd
Nov 29 12:40:47 hypervisor3 pacemakerd[989]:  error: pacemaker-attrd[1006] exited with status 102 (Not connected)
Nov 29 12:40:47 hypervisor3 pacemakerd[989]:  error: Connection to the CPG API failed: Library error (2)
Nov 29 12:40:47 hypervisor3 pacemakerd[989]:  crit: Lost connection to cluster layer, shutting down
Nov 29 12:40:47 hypervisor3 corosync[966]:  [SERV  ] Service engine unloaded: corosync cluster closed process group service v1.01
Nov 29 12:40:47 hypervisor3 corosync[966]:  [SERV  ] Service engine unloaded: corosync cluster quorum service v0.1
Nov 29 12:40:47 hypervisor3 corosync[966]:  [SERV  ] Service engine unloaded: corosync profile loading service
Nov 29 12:40:47 hypervisor3 corosync[966]:  [MAIN  ] Corosync Cluster Engine exiting normally
Nov 29 12:40:47 hypervisor3 pacemaker-schedulerd[1007]:  notice: Caught 'Terminated' signal
Nov 29 12:40:47 hypervisor3 systemd[1]: corosync.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Corosync Cluster Engine.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Network is Online.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Network.
Nov 29 12:40:47 hypervisor3 systemd[1]: netcf-transaction.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Rollback uncommitted netcf network config change transactions.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-networkd-wait-online.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Wait for Network to be Configured.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopping Network Name Resolution...
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-resolved.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Network Name Resolution.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopping Network Service...
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-networkd.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Network Service.
Nov 29 12:40:47 hypervisor3 systemd[1]: votp-config_ovs.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped "Configure OVS bridges and ports".
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopping Open vSwitch Forwarding Unit...
Nov 29 12:40:47 hypervisor3 ovs-ctl[5818]: Exiting ovs-vswitchd (368).
Nov 29 12:40:47 hypervisor3 kernel: handler230 (877) used greatest stack depth: 11040 bytes left
Nov 29 12:40:47 hypervisor3 systemd[1]: Requested transaction contradicts existing jobs: Transaction for systemd-networkd.service/start is destructive (shutdown.target has 'start' job queued, but 'stop' is included in transaction).
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-networkd.socket: Failed to queue service startup job (Maybe the service file is missing or not a non-template unit?): Transaction for systemd-networkd.service/start is destructive (shutdown.target has 'start' job queued, but 'stop' is included in transaction).
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-networkd.socket: Failed with result 'resources'.
Nov 29 12:40:47 hypervisor3 kernel: device vxlan_sys_4789 left promiscuous mode
Nov 29 12:40:47 hypervisor3 systemd[1]: ovs-vswitchd.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Open vSwitch Forwarding Unit.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopping Open vSwitch Database Unit...
Nov 29 12:40:47 hypervisor3 ovs-ctl[5956]: Exiting ovsdb-server (308).
Nov 29 12:40:47 hypervisor3 systemd[1]: ovsdb-server.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Open vSwitch Database Unit.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Network (Pre).
Nov 29 12:40:47 hypervisor3 systemd[1]: ip6tables.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped IPv6 Packet Filtering Framework.
Nov 29 12:40:47 hypervisor3 systemd[1]: iptables.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped IPv4 Packet Filtering Framework.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Basic System.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Paths.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-ask-password-console.path: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Dispatch Password Requests to Console Directory Watch.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-ask-password-wall.path: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Forward Password Requests to Wall Directory Watch.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Slices.
Nov 29 12:40:47 hypervisor3 systemd[1]: Removed slice User and Session Slice.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Sockets.
Nov 29 12:40:47 hypervisor3 systemd[1]: dbus.socket: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Closed D-Bus System Message Bus Socket.
Nov 29 12:40:47 hypervisor3 systemd[1]: libvirtd-admin.socket: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Closed Libvirt admin socket.
Nov 29 12:40:47 hypervisor3 systemd[1]: libvirtd-ro.socket: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Closed Libvirt local read-only socket.
Nov 29 12:40:47 hypervisor3 systemd[1]: libvirtd.socket: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Closed Libvirt local socket.
Nov 29 12:40:47 hypervisor3 systemd[1]: sshd.socket: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Closed sshd.socket.
Nov 29 12:40:47 hypervisor3 systemd[1]: virtlockd.socket: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Closed Virtual machine lock manager socket.
Nov 29 12:40:47 hypervisor3 systemd[1]: virtlogd-admin.socket: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Closed Virtual machine log manager socket.
Nov 29 12:40:47 hypervisor3 systemd[1]: virtlogd.socket: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Closed Virtual machine log manager socket.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target System Initialization.
Nov 29 12:40:47 hypervisor3 rngd[224]: [rdrand]: Shutting down
Nov 29 12:40:47 hypervisor3 rngd[224]: [jitter]: Shutting down
Nov 29 12:40:47 hypervisor3 rngd[224]: [jitter]: Closing thread 0
Nov 29 12:40:47 hypervisor3 rngd[224]: [jitter]: Closing thread 1
Nov 29 12:40:47 hypervisor3 rngd[224]: [jitter]: Closing thread 2
Nov 29 12:40:47 hypervisor3 rngd[224]: [jitter]: Closing thread 3
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopping Hardware RNG Entropy Gatherer Daemon...
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-sysctl.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Apply Kernel Variables.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-modules-load.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Load Kernel Modules.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopping Network Time Synchronization...
Nov 29 12:40:47 hypervisor3 systemd[1]: rngd.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Hardware RNG Entropy Gatherer Daemon.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-timesyncd.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Network Time Synchronization.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-tmpfiles-setup.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Create Volatile Files and Directories.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Local File Systems.
Nov 29 12:40:47 hypervisor3 systemd[1]: Unmounting Temporary Directory (/tmp)...
Nov 29 12:40:47 hypervisor3 systemd[1]: Unmounting /var/volatile...
Nov 29 12:40:47 hypervisor3 systemd[1]: tmp.mount: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Unmounted Temporary Directory (/tmp).
Nov 29 12:40:47 hypervisor3 systemd[1]: var-volatile.mount: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Unmounted /var/volatile.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Local File Systems (Pre).
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped target Swap.
Nov 29 12:40:47 hypervisor3 systemd[1]: Reached target Unmount All Filesystems.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-remount-fs.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Remount Root and Kernel File Systems.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-fsck-root.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped File System Check on Root Device.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-tmpfiles-setup-dev.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Stopped Create Static Device Nodes in /dev.
Nov 29 12:40:47 hypervisor3 systemd[1]: Reached target Shutdown.
Nov 29 12:40:47 hypervisor3 systemd[1]: Reached target Final Step.
Nov 29 12:40:47 hypervisor3 systemd[1]: systemd-reboot.service: Succeeded.
Nov 29 12:40:47 hypervisor3 systemd[1]: Started Reboot.
Nov 29 12:40:47 hypervisor3 systemd[1]: Reached target Reboot.
Nov 29 12:40:47 hypervisor3 systemd[1]: Shutting down.
Nov 29 12:40:48 hypervisor3 systemd-shutdown[1]: Syncing filesystems and block devices.
Nov 29 12:40:48 hypervisor3 systemd-shutdown[1]: Sending SIGTERM to remaining processes...
Nov 29 12:40:48 hypervisor3 pacemaker-execd[1005]:  notice: Caught 'Terminated' signal
Nov 29 12:40:48 hypervisor3 systemd-journald[166]: Journal stopped
```

**Comment by insatomcat** [comment]
see #63

**Comment by insatomcat** [comment]
closed by #63
